### PR TITLE
add volume snapshot support for IBM VPC CSI driver

### PR DESCRIPTION
**Pull request by dobsonj**
Version(s): 4.12+

Issue:
https://issues.redhat.com/browse/STOR-1060

Link to docs preview:

https://52657--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi.html#csi-drivers-supported_persistent-storage-csi

QE review:
- [x] QE has approved this change.

Additional information:

/label peer-review-needed
/cc @lpettyjo @chao007 @jsafrane
/hold for https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/46

### CODE REVIEW

**Comment by tmalove** [comment]
@dobsonj Will you add a preview link? Thanks!

**Comment by dobsonj** [comment]
/unhold
link added, prereq PR merged, QE approved

File: modules/persistent-storage-csi-drivers-supported.adoc [hunk 1]
```diff
@@ -42,6 +42,10 @@ The following table describes the CSI drivers that are installed with {product-t
 * Offline volume expansion: minimum required vSphere version is 6.7 Update 3 P06
 
 * Online volume expansion: minimum required vSphere version is 7.0 Update 2.
+
+3.
+
+* Does not support offline snapshots or resize, volume must be attached to a running pod.
```
Comment (lpettyjo) [inline modules/persistent-storage-csi-drivers-supported.adoc:48]: Small nit. This should be two sentences: "Does not support offline snapshots or resize. Volume must be attached to a running pod."

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: modules/persistent-storage-csi-drivers-supported.adoc (hunk 1 above)
Comment (dobsonj) [reply modules/persistent-storage-csi-drivers-supported.adoc]: Fixed, thanks!

**Comment by jsafrane** [comment]
lgtm

**Pull request merged**